src/typescript-service.ts

```diff
@@ -439,11 +438,8 @@ export class TypeScriptService {
 	}
 
 	protected async _workspaceSymbolDefinitelyTyped(params: WorkspaceSymbolParams): Promise<SymbolInformation[] | null> {
-		const rootUriParts = url.parse(this.rootUri);
-		if (!rootUriParts.pathname) {
-			return null;
-		}
-		const packageJsonUri = url.format({ ...rootUriParts, pathname: path_.posix.join(rootUriParts.pathname, 'package.json') });
+		const normRootUri = this.rootUri.endsWith('/') ? this.rootUri : this.rootUri + '/';
+		const packageJsonUri = normRootUri + 'package.json';
```

Comment:
I don't understand why this change is needed, `path.join()` handles trailing slashes:
```js
> require('path').posix.join('hello/', 'world')
'hello/world'
> require('path').posix.join('hello', 'world')
'hello/world'
> require('path').posix.join('hello', '/world')
'hello/world'
> require('path').posix.join('hello/', '/world')
'hello/world'
```

this change doesn't work if the root URI has a query or hash component